cypress/integration/sitemap-vrt/constants.ts

```diff
@@ -21,6 +21,8 @@ export const SITEMAP = [
   '/components/account-switcher/',
   '/components/aspect-ratio/',
   '/components/anchor/',
+  '/components/anchor/api',
```

Comment:
question: is there any way we can automate this?

Reply:
Probably, but not for this PR. A small node script could probably do this for us.